logger/sender_multi.go

```diff
@@ -59,9 +62,28 @@ func NewQueueMultiSender(ctx context.Context, workers, capacity int, senders ...
 }
 
 func (s *multiQueueSender) Send(m message.Composer) {
+	s.mu.Lock()
+	defer s.mu.Unlock()
```

Comment:
I think this can be an RLock

Reply:
how so? we are writing to the queue here